README.adoc

```diff
@@ -245,40 +245,39 @@ Create the `FaultToleranceTest` class in the
 include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
 ----
 
-The `@BeforeClass` annotation is placed on a method that executes before any of the test cases. In
+The `@BeforeClass` annotation is placed on a method that executes before any of the other test cases. In
```

Comment:
I think this can be reworded to the below to work around the passive voice: 
The `@BeforeClass` annotation places on a method that executes before any of the other test cases.
(Basically, the method has the `@BeforeClass` annotation.)

Reply:
This version makes it sound like the @BeforeClass annotation is placing something on a method. "The @BeforeClass annotation places on a method that executes before any of the other test cases." But don't we want to say that the annotation is the thing that is placed on the method?

Is the user placing the annotation on the method?
If so, then we could say, "Place the @BeforeClass annotation on a method that executes before any of the other test cases."

Reply:
No, we don't have the user to add line of codes one by one anymore. The user creates the whole test case class, and the text is to explain the key things in the class. I'd say, 
"The @BeforeClass annotation indicates that this method executes before any of the other test cases." what do you think?

Reply:
OK, that makes sense.

Let's go with your suggestion of, "The `@BeforeClass` annotation indicates that this method executes before any of the other test cases."